package-lock.json

```diff
@@ -1,8 +1,884 @@
 {
   "name": "website-dashboard",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
+      "name": "website-dashboard",
+      "version": "1.0.0",
+      "license": "ISC",
+      "dependencies": {
+        "puppeteer": "^12.0.1",
+        "puppeteer-core": "^12.0.1"
+      },
+      "devDependencies": {
+        "pre-commit": "^1.2.2",
+        "prettier": "2.3.1"
+      }
+    },
+    "node_modules/@types/node": {
+      "version": "17.0.20",
+      "resolved": "https://registry.npmjs.org/@types/node/-/node-17.0.20.tgz",
+      "integrity": "sha512-Q15Clj3lZSLnhVA6yKw1G7SQz46DeL9gO1TEgfK1OQGvMdQ6TUWmCeWf1QBUNkw2BDfV52i2YuYd9OF3ZwGhjw==",
+      "optional": true
+    },
+    "node_modules/@types/yauzl": {
+      "version": "2.9.2",
```

Comment:
Why are we changing package-lock.json, when no package is installed or changed

Reply:
It's automatically changed when running `npm install`

Reply:
The package-lock.json file is automatically generated and updated when running the npm install command. This file contains a detailed, versioned record of all the packages and dependencies that were installed in the project, including the exact versions of each package that were installed. This allows for consistency and reproducibility when installing dependencies on different machines or at different times. By - ChatGPT🤨

Reply:
reverted